docs/analytics/CAR-2013-01-002/index.md

```diff
@@ -8,8 +8,7 @@ analytic_type: Situational Awareness, TTP
 contributors: MITRE
 applicable_platforms: Windows
 ---
-
```

Comment:
Please confirm if the double `break` is what is desired over the newlines that were here - this is a change from the template file that's now been consistently applied.